src/pages/credits.jsx

```diff
@@ -3,8 +3,56 @@ import Image from 'next/image';
 import { NextSeo } from 'next-seo';
 import styles from '@/styles/modules/credits.module.scss';
 import ENV from '@/lib/constants/environmentVariables';
+import { withSession } from '@/lib/iron-session';
+import safeJsonStringify from 'safe-json-stringify';
+import prisma from '@/lib/db';
+import PropTypes from 'prop-types';
 
-function CreditsPage() {
+import ExternalLink from '@/components/shared/ExternalLink';
+import Contributor from '@/components/credits/Contributor';
+import CreditsMusic from '@/components/credits/CreditsMusic';
+
+const musicList = [
+  "XI12BK9VC3U", // SM64
+  "txe5UbkbqK0", // MK64
+  "87UysJxGiso", // PM64
+  "wvls_xrms3Q", // PMCS
+  "yZGQXG7Vy1I", // SMG
+  "V2-LgaXCnco", // SMS
+  "gZTAaJucWYY", // SMO
+  "2DRwPEvJrA8", // MK8
+]
+
+function getRandomCreditsMusic() {
+  return musicList[Math.floor(Math.random() * musicList.length)];
+}
+
+export const getServerSideProps = withSession(async ({ req }) => {
+  const username = req.session?.username;
+
+  if (!username) {
+    return {
+      props: {
+        accountInfo: {
+          name_on_riitag: "You",
+        }
+      }
+    };
+  }
+
+  const accountInfo = await prisma.user.findFirst({
+    where: {
+      username,
+    },
+    select: {
+      name_on_riitag: true,
+    },
+  });
+
+  return { props: { accountInfo: JSON.parse(safeJsonStringify(accountInfo)) } };
```

Comment:
Dou you need `safeJsonStringify` here? You could just use `accountinfo.name_on_riitag` directly. You can then use a fallback if no name was set.

Reply:
I just copied the code from another file here and forgot to clean it up